src/System.Runtime.Analyzers/VisualBasic/System.Runtime.VisualBasic.Analyzers.vbproj

```diff
@@ -116,8 +116,8 @@
     <Compile Include="BasicRethrowToPreserveStackDetails.Fixer.vb" />
     <Compile Include="BasicDoNotRaiseReservedExceptionTypes.vb" />
     <Compile Include="BasicDoNotRaiseReservedExceptionTypes.Fixer.vb" />
-    <Compile Include="BasicInitializeValueTypeStaticFieldsInline.vb" />
-    <Compile Include="BasicInitializeValueTypeStaticFieldsInline.Fixer.vb" />
+    <Compile Include="BasicInitializeStaticFieldsInline.vb" />
```

Comment:
+1. I think we can trim the 'Basic' prefix, it's confusing in any case for these C# tests. I'd suggest naming these classes [RuleName]Tests.cs|vb and [RuleName]FixerTests